src/main/java/com/denizenscript/denizencore/scripts/commands/core/DebugCommand.java

```diff
@@ -13,17 +17,17 @@ public class DebugCommand extends AbstractCommand implements Holdable {
 
     public DebugCommand() {
         setName("debug");
-        setSyntax("debug [<type>] [<message>] (name:<name>)");
-        setRequiredArguments(2, 3);
+        setSyntax("debug (<type>/{debug}) [<message>] (name:<name>) (format:<format>)");
+        setRequiredArguments(1, 3);

```

Comment:
I believe the new max is 4?

Reply:
Is it? I don't think anything uses both `name` and `format`?